### ISSUE TITLE
PHP notice: Undefined offset: 2

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15507]
### Summary
Log error:

```
[2021-03-07 12:13:01] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Transport/modTransportPackage.php : 165) PHP notice: Undefined offset: 2
[2021-03-07 12:16:27] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Transport/modTransportPackage.php : 707) PHP notice: A non well formed numeric value encountered
```

### Step to reproduce
Set system setting `log_level` to 4

Go to Extras -> Installer. Install for example Google Sitemap Generator or UltimateParent

### Observed behavior
```
[2021-03-07 12:13:01] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Transport/modTransportPackage.php : 165) PHP notice: Undefined offset: 2
[2021-03-07 12:16:27] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Transport/modTransportPackage.php : 707) PHP notice: A non well formed numeric value encountered
```

### Expected behavior
There should be no errors in the log

### Related issue(s)/PR(s)
NA

### Environment
MODX Revolution 3.0.0-alpha3 (git), PHP Version 7.4.12

**Comment by Mark-H** [comment]
Line 165:

``` php
list($this->version_major, $this->version_minor, $this->version_patch) = explode('.', $this->version);
```

Do those packages not have a patch version..?

**Comment by Ibochkarev** [comment]
Fixed #15791
